test/functional/test-url-replacements.js

```diff
@@ -20,6 +20,7 @@ import {urlReplacementsFor} from '../../src/url-replacements';
 import {markElementScheduledForTesting} from '../../src/custom-element';
 import {installCidService} from '../../src/service/cid-impl';
 import {installViewerService} from '../../src/service/viewer-impl';
+import {installActivityService} from '../../src/service/activity-impl';
```

Comment:
We need to do this slightly different. Check out how the cid service is installed in amp-analytics. And then here you only get a promise for it.

That way core does not depend on your service, but for people with amp-analytics it is available.